scylla-cdc-driver3/src/test/java/com/scylladb/cdc/cql/driver3/BaseScyllaIntegrationTest.java

```diff
@@ -0,0 +1,58 @@
+package com.scylladb.cdc.cql.driver3;
+
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.Session;
+import com.datastax.driver.core.schemabuilder.SchemaBuilder;
+import com.scylladb.cdc.cql.CQLConfiguration;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+
+import java.net.InetSocketAddress;
+import java.util.HashMap;
+import java.util.Properties;
+
+public class BaseScyllaIntegrationTest {
+    protected Session driverSession;
+    protected Cluster driverCluster;
+    protected Driver3Session librarySession;
+
+    @BeforeEach
+    public void beforeEach() {
+        Properties systemProperties = System.getProperties();
+        String hostname = systemProperties.getProperty("scylla.docker.hostname");
+        int port = Integer.parseInt(systemProperties.getProperty("scylla.docker.port"));
+
+        driverCluster = Cluster.builder()
+                .addContactPointsWithPorts(new InetSocketAddress(hostname, port))
+                .build();
+        driverSession = driverCluster.connect();
+
```

Comment:
Maybe we could `DROP KEYSPACE ks IF EXISTS` here, in case the previous cleanup didn't work?

Reply:
Applied your suggestion in amended commit.